node/src/accountant/mod.rs

```diff
@@ -5670,60 +5711,16 @@ mod tests {
                 }),
             }),
         };
-        let before = SystemTime::now();
 
         let result = subject.compute_financials(&request, context_id_expected);
 
-        let after = SystemTime::now();
-        let (computed_response, context_id) = UiFinancialsResponse::fmb(result).unwrap();
-        let extracted_receivable_ages = extract_ages_from_ui_receivable_accounts(
-            computed_response
-                .query_results_opt
-                .as_ref()
-                .unwrap()
-                .receivable_opt
-                .as_ref()
-                .unwrap(),
-        );
-        assert_eq!(context_id, context_id_expected);
-        assert_eq!(
-            computed_response,
-            UiFinancialsResponse {
-                stats_opt: None,
-                query_results_opt: Some(QueryResults {
-                    payable_opt: None,
-                    receivable_opt: Some(vec![UiReceivableAccount {
-                        wallet: make_wallet("efe4848").to_string(),
-                        age_s: extracted_receivable_ages[0],
-                        balance_gwei: 60
-                    }])
-                })
-            }
-        );
-        let time_needed_for_the_act_to_finish_its_job_in_secs =
-            (after.duration_since(before).unwrap().as_millis() / 1000 + 1) as u64;
-        assert!(
-            3333 <= extracted_receivable_ages[0]
-                && extracted_receivable_ages[0]
-                    <= 3333 + time_needed_for_the_act_to_finish_its_job_in_secs
-        );
-        let receivable_custom_query_params = receivable_custom_query_params_arc.lock().unwrap();
-        let actual_timestamp =
-            extract_timestamp_from_custom_query(&receivable_custom_query_params[0]);
-        assert_eq!(
-            *receivable_custom_query_params,
-            vec![CustomQuery::RangeQuery {
-                min_age_s: 2000,
-                max_age_s: 200000,
-                min_amount_gwei: 0,
-                max_amount_gwei: 150000000000,
-                timestamp: actual_timestamp
-            }]
-        );
-        assert!(before <= actual_timestamp && actual_timestamp <= after)
+        let (response, _) = UiFinancialsResponse::fmb(result).unwrap();
```

Comment:
You're replacing lots of expensive assertions with these two because you're already asserting the complete behavior of the `financials` command elsewhere, yes?